nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -79,10 +79,10 @@
     protected CoreSession session;
 
     /**
-     * @since 10.3
+     * @since 11.1
```

Comment:
Please keep the original `@since` value, the parameter is not new, you're only changing its type.

Reply:
yep,